tests/ui/test/specs/read-csv.spec.ts

```diff
@@ -27,15 +27,26 @@ test.describe('As a user I want to be able to read downloaded csv files', {
       expect(getColumnHeaders(filePath)).toEqual(["test_case", "some_value", "some_other_value"]);
   });
 
-  test('The data.csv file has correct row count', async () => {
-    expect(getRowCount(filePath)).toEqual(4);
+  [
+  { row: 4, column: 'test_case', expected: '4'  },
+  ].forEach(({ row, column, expected }) => {
+  test(`The data.csv file has correct value of ${expected} in row ${row} of 'test_case' ${column}`,{tag:'@Test'}, async () => {
```

Comment:
From what I can see these tests aren't "await"-ing anything so you don't need 'async' in the function declaration   ie. 
```suggestion
  test(`The data.csv file has correct value of ${expected} in row ${row} of 'test_case' ${column}`,{tag:'@Test'}, () => {
```
Either we are missing an 'await' or we can remove the 'async' declaration but believe it should be the latter from reading the code. Similarly there seems to be other tests in this file doing the same thing with making it as async function without awaiting a async method.

Reply:
Have updated. Let me know if I've missed anything